perllib/FixMyStreet/App/Controller/Admin.pm

```diff
@@ -1132,6 +1132,7 @@ sub user_edit : Path('user_edit') : Args(1) {
     $c->stash->{available_permissions} = $c->cobrand->available_permissions;
 
     $c->forward('fetch_all_bodies');
+    $c->forward('fetch_body_areas', [ $user->from_body ]) if $user->from_body;
```

Comment:
Ah yes, as how would it know which children to show. In which case, definitely shouldn't show the bit in the template if no children, as it will be confusing to superusers.